src/modules/missionHelper/settings.ts

```diff
@@ -47,11 +47,15 @@ export default (MODULE_ID: string): unknown => ({
         default: false,
         dependsOn: '.chances.normal',
     },
-    'multifunctionals.heavy_rescue_vehicles': {
-        type: 'toggle',
-        default: false,
-        dependsOn: '.vehicles.content',
-    },
+    ...(BUILD_LANG !== 'nl_NL'
+        ? {
+            'multifunctionals.heavy_rescue_vehicles': {
+                type: 'toggle',
+                default: false,
+                dependsOn: '.vehicles.content',
+            },
+        }
+        : null),
```

Comment:
@Suchty112 please check if that is correct. (Note: will be changed to "all languages except nl_NL")

Reply:
Currently Yes. can remain so. NL does not have a HLF, i.e. fire engine and rescue vehicle in one